types/react/test/elementAttributes.tsx

```diff
@@ -59,3 +59,40 @@ const testCases = [
         <img src="test" width={100} height={100} />
     </picture>,
 ];
+
+// Needed to check these HTML elements in event callbacks.
+// "Imported" from typescript's lib.dom.d.ts.
```

Comment:
I did because the react project does not compile against DOM:
```bash
~/projects/DefinitelyTyped/types/react$ jq ".compilerOptions.lib" tsconfig.json 
[
  "es6"
]
```
So, to be valid, the react project defines most HTML elements as empty interfaces [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4a7ecfdb673c4af91f69852a7a3f3f1266723a7a/types/react/global.d.ts#L32).

As they are empty interfaces, there is no way to easily test if they are correctly typed that I know of, so I added these inline interfaces adding a property on each HTML element, so I could check against that!

Reply:
Does this seem sensible @eps1lon?